src/fty_common_messagebus_malamute.cc

```diff
@@ -55,6 +55,9 @@ namespace messagebus {
                 }
             }
             else {
+                // Workaround for legacy message without metadata:
+                // For request/reply, need at least "from" meta data for reply management
+                message.metaData().emplace(Message::FROM, from);
```

Comment:
If we want to support "RAW" data by this way, we should also set "RAW" flag in the meta data

Reply:
You are right, it is better to add "RAW" flag in meta data. Take into account.